FIPS/pom.xml

```diff
@@ -2,9 +2,14 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>net.snowflake</groupId>
+  <parent>
+    <groupId>net.snowflake</groupId>
+    <artifactId>snowflake-jdbc-base</artifactId>
+    <version>3.14.3</version>
+    <relativePath>../base_pom.xml</relativePath>
```

Comment:
Will it work outside of the repo (for instance when someone fetches our JAR from maven central)?

Reply:
Should not be affected by this PR since it's a shaded JAR. And the dependencies in public_pom.xml remain the same.

Reply:
Please, do me a favour and perform a test:
1. `mvn install` everything
2. Remove `parent-pom.xml` from the local maven repo.
3. Create a new project (simplest POM) and add a new shaded POM as a dependency.
4. Try building such app.

Reply:
Discussed and tested in the backstage: verified, when after installing replacing pom in .m2 with public pom (simulating deployment to central).